pages/add-book.js

```diff
@@ -1,34 +1,58 @@
 import Head from "next/head";
-import Image from "next/image";
 import styles from "../styles/Home.module.css";
+import {useState} from "react";
```

Comment:
```suggestion
import { useState } from "react";
```
I'd recommend using Prettier for formatting. We can configure that somewhere if we want and someone wants to figure that out.